Docs/source/install/cmake.rst

```diff
@@ -147,10 +147,12 @@ Relative paths are also supported, e.g. ``-DWarpX_amrex_src=../amrex``.
 
 Or build against an AMReX feature branch of a colleague.
 Assuming your colleague pushed AMReX to ``https://github.com/WeiqunZhang/amrex/`` in a branch ``new-feature`` then pass to ``cmake`` the arguments: ``-DWarpX_amrex_repo=https://github.com/WeiqunZhang/amrex.git -DWarpX_amrex_branch=new-feature``.
```

Comment:
```suggestion
Assuming your colleague pushed AMReX to ``https://github.com/YourColleagueUserName/amrex/`` in a branch ``new-feature`` then pass to ``cmake`` the arguments: ``-DWarpX_amrex_repo=https://github.com/YourColleagueUserName/amrex.git -DWarpX_amrex_branch=new-feature``.
```
I would suggest this change

Reply:
But @WeiqunZhang is the most likely one :D

Reply:
As you prefer, then ;-)